addons/medical/ACE_Medical_Treatments.hpp

```diff
@@ -397,11 +428,27 @@ class ACE_Medical_Actions {
             callbackSuccess = QUOTE(DFUNC(actionPlaceInBodyBag));
             callbackFailure = "";
             callbackProgress = "";
+            treatmentType = "Others";
+            treatmentDisplay = "Placing in Bodybag";
             animationPatient = "";
             animationPatientUnconscious = "";
             itemConsumed = 1;
             litter[] = {};
         };
+        class SAMSplint: fieldDressing {
+            displayName = CSTRING(apply_samSplint);
+            displayNameProgress = CSTRING(applying_samSplint);
+            allowedSelections[] = {"hand_l", "hand_r", "leg_l", "leg_r"};
+            allowSelfTreatment = 0;
+            requiredMedic = 0;
+            items[] = {"ACE_samSplint"};
+            treatmentTime = 10;
+            callbackSuccess = QUOTE(DFUNC(treatmentSAMSplint));
+            treatmentType = "Others";
+            treatmentDisplay = "Applying SAM Splint";
+            condition = "[_this select 1, _this select 2] call ace_medical_fnc_canTreatSAMSplint";
```

Comment:
could just be `call ace_medical_fnc_canTreatSAMSplint` and let _this be passed implicitly. Rather than taking it apart (`select`) and then putting it back together (the array) just to take it apart again inside the function (`params`).

Reply:
or  `condition = QFUNC(canTreatSAMSplint)`;

Reply:
Roger, will have this fixed by tonight. Would it be better to use 'private _target = _this select 1' since the caller (0th item of _this array) or just accept all 4 items via 'params'?

Reply:
Used `QUOTE([ARR_2(_this select 1, _this select 2)] call FUNC(canTreatSAMSplint));`, similar to tourniquet and other treatments.